### DISCUSSION TITLE
about emqx rate limit  question

### DISCUSSION THREAD

**Discussion by jone-code** [emqx/emqx#9760] [Q&A]
if message sending rate exceeds config , 
1. Will the client get an error?   or  other ？？
 not on the documentation  (https://www.emqx.io/docs/zh/v5.0/reliability/rate-limit.html#%E6%A6%82%E5%BF%B5 )  
2.Whether there is an alarm can be seen in the dashboard

**Comment by lafirest** [comment]
@jone-code 

1. Only the `message_routing` will return an error `Quota exceeded` (Error Code 151) to the client, but other types do not belong to the MQTT spec, so there will no error for them
2. Now there isn't an alarm when covering the limit, but will print a debug message
